### PR TITLE
Vendor bump

### PR DESCRIPTION
**Pull request by gabemontero**
openshift/api vendor bump and switch from operatorstatus.Failing to Degraded

@openshift/sig-developer-experience fyi

### CODE REVIEW

File: pkg/operatorstatus/operatorstatus.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 	configv1 "github.com/openshift/api/config/v1"
 	configv1client "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
 
-	"github.com/openshift/cluster-samples-operator/pkg/apis/samples/v1"
+	v1 "github.com/openshift/cluster-samples-operator/pkg/apis/samples/v1"
```
Comment (gabemontero) [inline pkg/operatorstatus/operatorstatus.go]: golang 1.11 / vs code did this, not me

Comment (coreydaley) [reply pkg/operatorstatus/operatorstatus.go]: it should probably be samplesapiv1

File: test/e2e/cluster_samples_operator_test.go [hunk 2]
```diff
@@ -28,7 +28,7 @@ import (
 	samplesapi "github.com/openshift/cluster-samples-operator/pkg/apis/samples/v1"
 	sampopclient "github.com/openshift/cluster-samples-operator/pkg/client"
 	sampleclientv1 "github.com/openshift/cluster-samples-operator/pkg/generated/clientset/versioned"
-	"github.com/openshift/cluster-samples-operator/pkg/operatorstatus"
+	operator "github.com/openshift/cluster-samples-operator/pkg/operatorstatus"
```
Comment (coreydaley) [inline test/e2e/cluster_samples_operator_test.go]: This was changed from operatorstatus to operator, but there were no other changes? seems suspicious ...

Comment (coreydaley) [reply test/e2e/cluster_samples_operator_test.go]: s/samplesapi/samplesapiv1/
s/sampopclient/samplesopclient/

Comment (gabemontero) [reply test/e2e/cluster_samples_operator_test.go]: that also was a golang 1.11 / vscode thing

Comment (coreydaley) [reply test/e2e/cluster_samples_operator_test.go]: there might be a setting that you can change to not do that.

**Comment by gabemontero** [comment]
`Unauthorized` in image eco

/test e2e-aws-image-ecosystem

File: test/e2e/cluster_samples_operator_test.go (hunk 2 above)
Comment (gabemontero) [reply test/e2e/cluster_samples_operator_test.go]: good point @coreydaley ... I'll hunt for that next time

given the current set of green tests I'm thinking of eschewing generating a new commit and risking flakes :-)

**Comment by gabemontero** [comment]
bump @adambkaplan

last chance to punt on this, or shall we merge?

**Comment by adambkaplan** [comment]
/lgtm

We can address issues around imports/vscode in 4.2

**Comment by gabemontero** [comment]
aws vpc pain on last e2e-aws-operator

**Pull request merged**